docs/cron-workflows.md

```diff
@@ -103,6 +104,33 @@ For example, with timezone set at `America/Los_Angeles`, we have daylight saving
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+### Stopping a `CronWorkflow`
+
+> v3.6 and after
```

Comment:
We now have versioned docs so I am not sure if this is needed. @jmeridth and others - any thoughts?

Reply:
We did talk about this in a Contributor Meeting before we even had versioned docs, [in August I think?](https://docs.google.com/document/d/1tznN5LB-KrNkC6dmeIzpKfuyvZWgCJpRuS-E84zseC8/edit#heading=h.qogh82i9yij0), but it's not written in the meeting notes/minutes. The [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) says to use notation for future versions.

But more importantly, users assume a feature is usable unless it says otherwise. We've had that before versioned docs (such as [with `deleteDelayDuration`](https://github.com/argoproj/argo-workflows/pull/11445) and others) and also after versioned docs -- the markdown title & description feature had [several users](https://github.com/argoproj/argo-workflows/pull/12582#issuecomment-1944824694) asking about it on Slack and that was the primary reason I had mentioned my docs PR on [the contributors channel](https://cloud-native.slack.com/archives/C0510EUH90V/p1708533114288009).
So user behavior means this is 100% needed